#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

PKG = "jsk_topic_tools"

gen = ParameterGenerator()

gen.add("update_rate", double_t, 0, "Desired publish rate of throttled messages", 1.0, 0.0, 1000.0)
gen.add("duration_message_valid", double_t, 0, "Timeout duration of subscribed message", 1.0, 0.0, 1000.0)

exit(gen.generate(PKG, PKG, "ConstantRateThrottle"))
